IntelliCommand/Services/CommandScopeService.cs

```diff
@@ -175,11 +175,13 @@ private void ParseFile(string filePath)
                     
                     var name = xNameAttribute.Value;
 
-                    if (!this.scopeDefinitions.ContainsKey(id))
-                    {
-                        this.scopeDefinitions.Add(id, name);
-                    }
+//in old version:
+//if some scope's name differs in default VS settings and current user settings, assertion fails.
+//e.g. <Scope Name="XAML Designer" ID="{E9B8485C-1217-4277-9ED6-C825A5AC1968}"/> is such name
+//it is "XAML UI Designer" in default settings files, but Visual Studio Changes it to "XAML Designer" in user's CurentSettings.vssettings
+                    this.scopeDefinitions[id] = name;

```

Comment:
Ok, I will.
Still, I think these changes are relevant. It's common practice to load default settings first and them with user settings.
But one thing i can't understand. Why parse all settings files you can get? 
E.g. among the files parsed were backup copy of vssettings file in Settings directory (just copied manually), exported settings file in another directory and I don't know what else. I don't think I want to parse all those files, really. Altho this particular extension has little place to behave weird based on this, I have doubts Visual Studio loads setting this way. I think it combines all settings into CurrentSettings.vssettings anyway, so parsing only the last file should be ok. At least, I did just that and everything is just fine.
```
if (ErrorHandler.Succeeded(settingFilesCollection.GetSettingsFile(**count - 1**, out fileInfo)))
                {
                    string filePath;
                    if (ErrorHandler.Succeeded(fileInfo.GetFilePath(out filePath)))
                    {                    
                        this.ParseFile(filePath);
                    }
                }
```